app/services/trainees/create_from_csv_row.rb

```diff
@@ -105,15 +109,15 @@ def last_name
     end
 
     def itt_start_date
-      csv_row["Course ITT start date"]
+      csv_row["Course ITT start date"].presence&.to_date
     end
 
     def itt_end_date
-      csv_row["Course Expected End Date"]
+      csv_row["Course ITT end date"].presence&.to_date
```

Comment:
Given [this upcoming ticket](https://trello.com/c/2PJ30OIv/629-update-itt-end-date-to-expected-end-date), feels like this is going in the wrong direction?

Reply:
Plus it's causing test failures ;-)